.github/scripts/test_compatibility.sh

```diff
@@ -0,0 +1,68 @@
+# Testing compatibility
```

Comment:
nit: rename to `test_torch_version_torchao_version_compatibility.sh` to be specific